backvendor/glide/glide.go

```diff
@@ -0,0 +1,70 @@
+package glide
```

Comment:
Would it be better for this to be in backvendor/internal/glide if we're making a new package of it?

Reply:
I don't mind. `pkg` is also common. I just didn't do it because so far it is not a multi-binary project. Let me know if you want `internal`.

Reply:
Ok, let's keep it like this for now.